test/unit/test_generate_execute.py

```diff
@@ -47,7 +47,7 @@ class GenerateExecuteTest(unittest.TestCase):
             'test_directory': '__splitNjoin-generated-tests',
         },
         'failing' : {
-            'config_file': os.path.join(test_data_dir, 'failingApp', 'tkltest_config.toml'),
+            'config_file': os.path.join(test_data_dir, 'failing', 'tkltest_config.toml'),
```

Comment:
why did you change the name of the application?

Reply:
there was a convention that the app name= app dir.